0021-plugin-rpc-api/0021-plugin-rpc-api.md

```diff
@@ -43,181 +66,3 @@ Content-Type: application/json
 
 true
 ```
-
-### Send Request (ILQP)
```

Comment:
That's the beauty of it :) we define one API with two transports (in-process JS, and http), so we don't have to duplicate the same list of methods in both rfc 4 and 21.

In fact, I was thinking the http transport is the more generic one, so instead of rfc 21 referring to rfc 4, rfc 4 should refer to 21. So then rfc 21 gets all the text from rfc 4, just with 'http response body' instead of 'Promise' everywhere, and then rfc4 becomes a very short description of how the LPI should be adapted *from* http transport *to* JS-specific in-process function calls. It might feel like a big change in how we document things but I think defining a http api as the primary boundary between one connector and the next would make Interledger more accessible for non-JS developers.